test/setup_mongo_node.js

```diff
@@ -0,0 +1,28 @@
+var DB = require('mongodb').Db;
```

Comment:
This file doesn't look to get called by anything, does it? If it's intended to be called by mongo as a setup step before running the tests, consider wrapping it in a function and exporting that as `before`, and stop exiting the process at the end.

Reply:
It seemed to be necessary, but then I removed the setup clause, and forgot to remove the file.